### PR TITLE
Fix/styling

### PR DESCRIPTION
**Pull request by kristenmb**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
Adjusted styling across the app, adding media queries and adjust data display on multiple components/displays

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
- I was unable to solve a styling challenge with the mobile view of the calendar -- if we can find a way to change the calendar view to week or day view only on mobile screen size it will be much more user friendly. 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Curatour</title>
```
Comment (DrewBradley) [inline public/index.html:29]: YAS!!!

File: src/components/agenda/Agenda.css [hunk 2]
```diff
@@ -55,17 +66,16 @@
...
   margin: .25rem 0 .25rem 0;
-
+  color: black;
 }
 
 .agenda-item {
     display: flex;
     flex-direction: column;
-    width: 100%;
+    width: 40vw;
```
Comment (DrewBradley) [inline src/components/agenda/Agenda.css:75]: Smart! Should we put a 'max-width' on this for when the app is opened on a larger screen?

File: src/components/agenda/Agenda.css [hunk 3]
```diff
@@ -77,9 +87,40 @@
...
+
+  .agenda-description {
+    width: 80vw;
+  }
+
+  .input-wrapper {
+    width: 80vw;
+    justify-content: space-evenly;
+  }
+}
```
Comment (DrewBradley) [inline src/components/agenda/Agenda.css:126]: Thank you for added responsive units consistently!

**Review by DrewBradley — APPROVED** [review]
Wow! This was a huge lift! Great work! Everything looks a lot cleaner now!

**Review by kevinhartmann23 — APPROVED** [review]
You are a champion, that you for taking the time to solidify the style and bring it back to marching our Miro board!!

File: public/index.html [hunk 4]
```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
```
Comment (richardltyler) [inline public/index.html:3]: I was just thinking about needing to do this. Thanks!

File: src/components/calendar/CalendarWrapper.js [hunk 5]
```diff
@@ -25,12 +26,13 @@ export const CalendarWrapper = styled.div`
   }
 
   .fc table, td, th {
-    border: 2px solid #e2992f;
-    color: #e2992f;
+    border: 2px solid #fab856;
+    color: #fab856;
```
Comment (richardltyler) [inline src/components/calendar/CalendarWrapper.js:30]: I like the new color!

**Review by richardltyler — APPROVED** [review]
Woah this is a lot! Thanks for getting so much done!

**Pull request merged**